operationalGuide.md

```diff
@@ -0,0 +1,174 @@
+
+Please note that the three-tiered approach does not reflect the three card types, but different use cases as described below.
+
+![API Structure](https://github.com/swissfintechinnovations/ca-card/assets/116151702/f93b82c3-eba7-42db-9aac-f7e34037f936)
+
+## Name patterns for parameters
+
+### Differentiation between *code / *reference / *id
+The parameter names allow an easy recognition of the type of underlying identifier. The three following words are used as part of the parameter names for distinction:
+
+* *Id: technical identifier for an instance of a resource, is always in the uuid format; is used for direct access to a resource instance in an API endpoint
+* *Reference: functional identifier for an instance of a resource that can also be known by an end customer, is a generic string without a standardized format 
+* *Code: identifier for an instance of a resource, can be mapped to a name in a finite set of values, is a generic string without a standardized format
+
+
+# Card API Level 1
+
+## Model
+![Card-API_Level1](https://github.com/swissfintechinnovations/ca-card/assets/141315380/7ec14f1a-7fa8-4cdb-bca2-8f806f0e843c)
+
+### “cards” resource
+The term “cards” refers to both physical and virtual credit or debit cards issued by banks or financial institutions. Physical cards are the traditional plastic cards that holders carry with them. Virtual cards, on the other hand, only exist in digital form and are often used for online payments. These virtual cards have a card number, expiration date and security code like physical cards.
+### “card-tokens” resource
+The term “card tokens” refers to the DPAN (Digital Primary Account Number) of a credit or debit card that is stored in digital wallets, wearables or with merchants (e.g. for click-to-pay). The “card token” serves as a replacement for the PAN and is used for transactions instead of the real card number. This reduces the risk of data misuse or fraud, as the sensitive card data is not transmitted directly.
+### “transactions” resource
+The term “transactions” refers to payments made with physical or virtual cards or with card tokens. These transactions are displayed on the cardholder's account and include all authorized and recorded payments entered by the cardholder.
+
+Transactions can include various types of payments, such as retail purchases, online payments, transfers between accounts, etc. Each transaction is authorized and recorded, which means that it has been approved by the cardholder and the payment amount has been debited from the cardholder's account.
+
+## Use Cases
+| Identifier | Description |
+| -------- | ------- |
+| PFM/BFM | Aggregation of transactions for cards from different issuers to make evaluations on spending behavior, budget, ecological footprint or more.<br>Limitation: Account-based transactions are not taken into account.<br>Challenge with these endpoints: incomplete view of card account transactions. |
```

Comment:
- What is there the idea? Only show the cards in general (product, masked PAN, expiry date) , or also details about the cards (e.g. limits, multi account, card settings, etc.)?
- Againt client, transactions normally will be shown on level FPAN, not on Token level. Reasons: Authorizsation ist on level FPAN; for bank client more comprehensible

Reply:
- For level 1, only the information can be shown which are part of the data model.
- The data user can decide how the transactions are shown. The API provides for every transaction the card id. Token id is also available if it was a token-based transaction.

Reply:
should be discussed laterly in a working group

Reply:
Clarified that fpan / token_id is part of the transaction data of the schemes. Some potential data providers might not have this information currently so it could be missing in the data they provide.